packages/linux/package.mk

```diff
@@ -44,6 +44,13 @@ case "$LINUX" in
     PKG_SOURCE_DIR="$PKG_NAME-amlogic-$PKG_VERSION*"
     PKG_PATCH_DIRS="amlogic-3.14"
     ;;
+  amlogic-3.14-s9xx)
```

Comment:
Please revert that. We don't want more fragmentation for AML, we need more collaboration. All necessary updates should be made to our linux-amlogic fork.